identity/webapp/src/middleware/error-handler.ts

```diff
@@ -4,7 +4,7 @@ import { ServerError } from '../utility/errors/server-error';
 import { ConflictError } from '../utility/errors/conflict';
 import { ApiError } from '../utility/errors/api-error';
 import { NotFound } from '../utility/errors/not-found';
-import {BruteForceBlocked} from "../utility/errors/rate-limit";
+import {BruteForceBlocked} from '../utility/errors/rate-limit';
```

Comment:
```suggestion
import { BruteForceBlocked } from '../utility/errors/rate-limit';
```

Reply:
Actually the path looks to be incorrect:

```suggestion
import { BruteForceBlocked } from '../utility/errors/brute-force-blocked';
```